app/routes/docs.contributing.github/route.mdx

```diff
@@ -0,0 +1,122 @@
+---
+handle:
+  breadcrumb: GitHub
+---
+
+import copyUrl from './copy-url.png'
+import githubFork from './github-fork.png'
+import githubImage from './github.png'
+import newFork from './new-fork.png'
+
+# Prerequisites
+
+You will need to install Git on your development machine. Check out [the official installation instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+We recommend checking out [this guide](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) on adding an SSH key to your github account.
+
+# GitHub
+
+GCN consists of several open-source projects grouped into the [nasa-gcn](https://github.com/nasa-gcn) GitHub organization. These include:
+
+- [nasa-gcn/gcn.nasa.gov](https://github.com/nasa-gcn/gcn.nasa.gov): This web site
+- [nasa-gcn/gcn-schema](https://github.com/nasa-gcn/gcn.nasa.gov): The GCN Notices unified schema
+- [nasa-gcn/gcn-kafka-python](https://github.com/nasa-gcn/gcn-kafka-python), [nasa-gcn/gcn-kafka-js](https://github.com/nasa-gcn/gcn-kafka-js): Kafka client libraries
+  ...
+
+To contribute to any project in the [nasa-gcn](https://github.com/nasa-gcn) GitHub organization, follow these steps:
+
+- [Forking a Repository](#forking-a-repository)
+
+- [Cloning a Repository](#cloning-a-repository)
+
+- [Creating a Branch](#creating-a-branch)
+
+- [Pull Requests](#pull-requests)
+
```

Comment:
- [ ] Switch the order of these two sections.
- [ ] Make the prerequisites section a second-level heading (`##`).
- [ ] Make sure that the headings are all the same kinds of phrase (gerund phrases are fine). So for example, rename "Prerequisites" to something like "Setting up Git on your computer"